src/main/java/com/example/springresttask/repository/EmailRepositoryImpl.java

```diff
@@ -0,0 +1,121 @@
+package com.example.springresttask.repository;
+
+import com.example.springresttask.domain.Email;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Slf4j
+public class EmailRepositoryImpl implements EmailRepository {
+    
+    @Value("${spring.datasource.url}")
+    private String URL;
+    @Value("${spring.datasource.username}")
+    private String USERNAME;
+    @Value("${spring.datasource.password}")
+    private String PASSWORD;
+
+    private static Connection connection;
+    private ResultSet resultSet;
+
+    @PostConstruct
+    void init() {
+        try {
+            connection = DriverManager.getConnection(URL, USERNAME, PASSWORD);
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+    }
+
+    @Override
+    public List<Email> findAllByPendingEmail() {
+
+        List<Email> emails = new ArrayList<>();
+        try {
+            //todo:can it be shorter?
+            String SQL = "select id,recipient_name,email_subject,email_body,delivery_date," +
```

Comment:
1)
Your SQL variable is acting like a constant.
Either make it a normal variable or make it as a constant.

Check [Java code conventions](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf).

2) 
If you wish to format your SQL string in better way I propose to use multiline string:

```
String sql = """
    SELECT id, recipient_name, ...
    FROM emails
    WHERE is_sent IS FALSE;
""";
```

3) 
Table should be named `email` instead of `Email`.